modules/client-handler/src/main/java/org/apache/ignite/client/handler/ClientInboundMessageHandler.java

```diff
@@ -681,16 +684,25 @@ private void processOperation(ChannelHandlerContext ctx, ClientMessageUnpacker i
                 return null;
 
             case ClientOp.TABLES_GET:
-                return ClientTablesGetRequest.process(out, igniteTables).thenRun(() -> {
-                    out.meta(clockService.current());
-                });
+                if (clientContext.hasFeature(ServerProtocolBitmaskFeature.TABLE_GET_REQS_USE_QUALIFIED_NAME)) {
```

Comment:
I think a cleaner approach is to introduce separate op codes: `ClientOp.TABLES_GET_QUALIFIED` and `TABLE_GET_QUALIFIED`.